DESIGN.md

```diff
@@ -0,0 +1,60 @@
+# Bulk Data Extract (BDE) Processor design
+
+## User requirements
+
+-  The _user interface_ is a read-only PostgreSQL database in AWS RDS, with
+   Azure authentication
+-  _Accessible_ to all
+   [current BDE users](https://github.com/linz/ds-infra-team/wiki/BDE-users)
+-  Process BDE files
+   _[every day](https://toitutewhenua.atlassian.net/wiki/spaces/GEOD/pages/86418416/BDE+Processor+Product)
+   except Sunday_
+-  Optional _post-processing_ pulled in from separate internal repository
+-  All changes for a single day are imported within a _single transaction_
+-  _Immediate notification_ of completed import (like the legacy BDE Processor
+   SNS topic)
+
+## Technical requirements
+
+-  Thorough testing of every part of the process before merging any pull
+   requests
+-  Separate, managed accounts for infrastructure testing and production
+-  Trivial deployment
+
+## Open questions
+
+-  Can we give read access to the BDE data to the entire organisation? The
```

Comment:
For the organisation: yes, as long as it didn't affect processing or publishing to LDS, like performance etc. This is what the replica/analytics stuff is designed to do).

For outside the org: yes and no. We can make some tables publically accessible, some are available under a creative commons license on the LDS adn some under a more restrictive customer license. So we would have to effectively recreate that.